catalog/spatial/geocoding/spatial-geocoding-lucene/src/main/java/org/codice/ddf/spatial/geocoding/query/GeoNamesQueryLuceneDirectoryIndex.java

```diff
@@ -122,10 +128,16 @@ static Shape getShape(String location) throws ParseException {
   @Override
   public Optional<String> getCountryCode(String wktLocation, int radius)
       throws GeoEntryQueryException, ParseException {
```

Comment:
Looks like this ParseException can be removed. Do we need the GeoEntriesQueryException anymore either?

Reply:
We can get rid of ParseException, but not GeoEntriesQueryException